### ISSUE TITLE
Implement Neural Model Architecture (ParserMLP)

### ISSUE THREAD

**Issue by Emilio25232** [Emilio25232/nlu-p2#8]
### Description
Define the MLP-based dependency parser (embeddings + dense layers + softmax).

### Tasks
- [ ] Create `__init__` storing hyperparameters
- [ ] Define Keras Input layer
- [ ] Add Embedding layers (FORM + UPOS)
- [ ] Concatenate embeddings
- [ ] Add Dense hidden layers (ReLU)
- [ ] Add softmax output(s) for transitions and deprels
- [ ] Compile the model and store in `self.model`
